arch/c4_context.md

```diff
@@ -6,40 +6,70 @@ All production Systems (External and Radix) and Users that are commonly found on
 
 ![c4_context.puml](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/radixdlt/docs/arch/arch/c4_context.puml)
 
+# Universes
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+From an architectural point of view, a Universe identifies and overlay network on top of Internet or in a LAN.
+Core/Boot nodes can be configured to be part of a specific universe.
+
+# Atoms and Particles
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+On an Architectural level an Atom is totally ordered sequence of Transactions (Particles) that must be validated as a unit and committed to the Ledger (DB) atomically. 
+
+The minimal unit of transactional data transferred over wire (sync/gossip/WebSocket) is thus an Atom.
+
 # Shards and Chunks
 
-Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/), shards enable Radix Networks to scale infinitely.
-This is achieved by splitting up global state (Ledger) to different nodes. Each node can be part of one or more Shards slices - called Chunks.
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+Sharding enable Radix Networks to have unlimited scalability.
+This is achieved by splitting up global state (Ledger) to different nodes in a deterministic way.
+The goal with this sharding scheme is to be able to tell where certain Particle (transaction) goes just by looking at it, so that we can check double spends (and other attack vectors) efficiently.
```

Comment:
> The goal with this sharding scheme is to be able to tell where certain Particle (transaction) goes just by looking at it, so that we can check double spends (and other attack vectors) efficiently

The key of this sharding scheme is to be able to tell where in the shard space a certain state transition (i.e. Particle, transaction) is located, rendering double-spends and other attack vectors on "opposite sides" of the network impossible because e.g. a certain spend can only come from a single place i.e. shard.